src/main/java/com/microfocus/application/automation/tools/run/PcBuilder.java

```diff
@@ -262,13 +262,25 @@ private Testsuites run(PcClient pcClient, Run<?, ?> build)
         boolean trendReportReady = false;
         try {
             runId = pcClient.startRun();
-            testName = pcClient.getTestName();
+            try {
```

Comment:
Wouldn't it be better to split the `try catch` to functions to split functionality?